eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Data.hs

```diff
@@ -93,8 +93,6 @@ instance FromCBOR (Annotator Plutus.Data) where
 instance ToCBOR Plutus.Data where
   toCBOR = Cborg.encode
 
-deriving anyclass instance NoThunks Plutus.BuiltinByteString
```

Comment:
It seems `Plutus.BuiltinByteString` no longer has a Generic instance to make this work. But it also does not appear to be needed anymore.